waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/usecase/claim/SilentClaimStrategy.kt

```diff
@@ -47,10 +50,12 @@ class SilentClaimStrategy(
         val issuerDid = WalletCredential.parseIssuerDid(credential, manifest) ?: "n/a"
         val type = credentialTypeSeeker.get(credential)
         val egfUri = "test"
-        //TODO: improve for same issuer - type values
-        if (validateIssuer(issuerDid, type, egfUri)) {
+        if (ConfigManager.getConfig<TrustConfig>().issuersRecord.baseUrl.startsWith("http")) {
```

Comment:
not sure I understand the meaning of this check, could you please elaborate on it